src/Compiler/Driver/CompilerDiagnostics.fs

```diff
@@ -320,6 +321,7 @@ type Exception with
         | BadEventTransformation _ -> 91
         | HashLoadedScriptConsideredSource _ -> 92
         | UnresolvedConversionOperator _ -> 93
+        | DefinitionsInSigAndImplNotCompatibleAbbreviationsDiffer _ -> 318
         | ArgumentsInSigAndImplMismatch _ -> 3218
```

Comment:
Would it be possible to try to keep things at least mostly sorted? It looks like the most of the errors are sorted by number here.

Reply:
Fine, sorted but if the build now breaks I'm reverting this.